src/pages/EventDetails/components/EventRenderer.tsx

```diff
@@ -180,12 +180,26 @@ const EventRenderer = ({ data, preview = false }: EventRendererProps) => {
             variant='outlined'>
             Meld deg av
           </VerifyDialog>
-        ) : (
-          isFuture(startDate) && (
-            <Alert severity='info' variant='outlined'>
-              Avmeldingsfristen er passert
+        ) : isFuture(sub(Date.parse(data.start_date), { hours: 2 })) ? (
+          <>
+            <VerifyDialog
+              closeText='Avbryt'
+              confirmText='Ja, jeg er sikker'
+              contentText={`Om du melder deg av nå vil du ikke kunne melde deg på igjen. I tillegg vil du få en prikk.`}
+              fullWidth
+              onConfirm={signOff}
+              titleText='Er du sikker?'
+              variant='outlined'>
+              Meld deg av
+            </VerifyDialog>
+            <Alert severity='warning' variant='outlined'>
```

Comment:
```suggestion
            <Alert severity='info' variant='outlined'>
```

Dette er informasjon siden bruker ikke har trykket på knappen